.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         build_command: [
           'macos_build FluentUITestApp-macOS Release build',
           'macos_build FluentUITestApp-macOS Debug build -resultBundlePath TestResultsMac test -destination "platform=macOS,arch=x86_64"',
-          'ios_simulator_build Demo.Development Debug build test -resultBundlePath TestResultsiOS -destination "platform=iOS Simulator,name=iPhone 14 Pro"',
+          'ios_simulator_build Demo.Development Debug build test -resultBundlePath TestResultsiOS -destination "platform=iOS Simulator,name=iPhone 16 Pro"',
```

Comment:
Could move the string to a variable, I do that elsewhere if you want a pointer

Reply:
Gonna leave it as-is for now, but let's chat offline for future updates